# 7daf51d797 	[illumos-gate merge]
# tools 62ad3b88bdc2b2e61372dfceaf22cf2e9fbd1b32 -> master 7daf51d797cfbba11b7fbc78631da0c8b3f275a6

manifest | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/manifest b/manifest
index c8e6ff89b8..73a2b8e88e 100644
--- a/manifest
+++ b/manifest
@@ -1969,6 +1969,7 @@ f platform/i86pc/kernel/drv/amd64/amd_iommu 0755 root sys
 f platform/i86pc/kernel/drv/amd64/cpc 0755 root sys
 f platform/i86pc/kernel/drv/amd64/cpudrv 0755 root sys
 f platform/i86pc/kernel/drv/amd64/dr 0755 root sys
+f platform/i86pc/kernel/drv/amd64/efi 0755 root sys
 f platform/i86pc/kernel/drv/amd64/fipe 0755 root sys
 f platform/i86pc/kernel/drv/amd64/ioat 0755 root sys
 f platform/i86pc/kernel/drv/amd64/isa 0755 root sys
@@ -1981,6 +1982,7 @@ f platform/i86pc/kernel/drv/amd64/rootnex 0755 root sys
 f platform/i86pc/kernel/drv/amd_iommu.conf 0644 root sys
 f platform/i86pc/kernel/drv/cpc.conf 0644 root sys
 f platform/i86pc/kernel/drv/dr.conf 0644 root sys
+f platform/i86pc/kernel/drv/efi.conf 0644 root sys
 f platform/i86pc/kernel/drv/fipe.conf 0644 root sys
 f platform/i86pc/kernel/drv/ioat.conf 0644 root sys
 f platform/i86pc/kernel/drv/pci-ide.conf 0644 root sys
@@ -19272,6 +19274,7 @@ f usr/share/man/man4d/dnet.4d 0444 root bin
 f usr/share/man/man4d/dtrace.4d 0444 root bin
 f usr/share/man/man4d/e1000g.4d 0444 root bin
 f usr/share/man/man4d/ee100x.4d 0444 root bin
+f usr/share/man/man4d/efi.4d 0444 root bin
 f usr/share/man/man4d/ehci.4d 0444 root bin
 f usr/share/man/man4d/ena.4d 0444 root bin
 f usr/share/man/man4d/fasttrap.4d 0444 root bin
